.github/workflows/update-metadata-glsa.yml

```diff
@@ -0,0 +1,58 @@
+name: Keep GLSA metadata updated
+on:
+  schedule:
+    - cron:  '0 7 1 * 1'
+  workflow_dispatch:
+
+jobs:
+  keep-glsa-metadata-updated:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout portage-stable
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: ./portage-stable
+      - name: Checkout Gentoo
+        uses: actions/checkout@v3
+        with:
+          repository: gentoo-mirror/gentoo
+          path: gentoo
+          fetch-depth: 1
```

Comment:
you mentioned this is the default, is it needed here?

Reply:
removed fetch-depth